org.gnunet.Messenger.json

```
@@ -0,0 +1,148 @@
+{
+  "app-id": "org.gnunet.Messenger",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "messenger-gtk",
+  "finish-args": [
+    "--share=ipc",
+	"--socket=wayland",
+	"--socket=fallback-x11",
+	"--share=network",
+	"--socket=pulseaudio",
+	"--device=dri",
+
+	/* Home directory access to share files */
+	"--filesystem=home",
+	
+	/* Webcam access to scan QR codes */
+	"--device=all"
+  ],
+  "modules": [
+    {
+	  "name": "recutils",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://ftp.gnu.org/gnu/recutils/recutils-1.8.tar.gz",
+		  "sha512": "607284f3e0f067c4fbbd239ff93fdadcf0864d45cb7d27c955a79595778cd76531043f6896f00345fc74853c5fd63bd2379deb4f7d26d49cf515ae46a2dd937c"
+		}
+	  ]
+	},
+	{
+	  "name": "jansson",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://digip.org/jansson/releases/jansson-2.13.tar.gz",
+		  "sha512": "79e26745e257118d4860cbfd57e250834c0766523d7f5c3d9e34438805f9c3064a5890ba3108292e5f2cb24d77a77049ffddd17332c65c293b70fb91869b67e6"
+		}
+	  ]
+	},
+	{
+	  "name": "libsodium",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://download.libsodium.org/libsodium/releases/libsodium-1.0.18-stable.tar.gz",
+		  "sha512": "0552c5bdd6521a7af4c54f3fe5926e66db71b7b0bc83f9b33a06538f1afa18019a012fa3da562bbcf780dfadb2595aaf35806a77729062087920eb5f844dfd54"
+		}
+	  ]
+	},
+    {
+	  "name": "gnunet",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://ftpmirror.gnu.org/gnunet/gnunet-0.16.3.tar.gz",
+		  "sha512": "fad8fcdb4b8058f17aef13a6176d8f19ab88d77fc5fcc709a4bbda463b2217ed2342f3e858c634114b9e4cf8651082af19ca1d490b1dce5e4302c421e1f88664"
+		}
+	  ]
+	},
+	{
+	  "name": "libgnunetchat",
+	  "buildsystem": "simple",
+	  "build-commands": [
+	    "make release",
+		"make INSTALL_DIR='/app/' install"
+	  ],
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://ftpmirror.gnu.org/gnunet/libgnunetchat-0.1.0.tar.gz",
+		  "sha512": "3839336bfdecdf538b79a6c24ceef6170d221c43f70ce11a43466be1fe2d7a22baf0ab5dd9794baa0964674842516ce2f8378f2b03b378adc2ec318c1e88c277"
+		}
+	  ]
+	},
+	{
+	  "name": "libqrencode",
+	  "buildsystem": "autotools",
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.gz",
+		  "sha512": "209bb656ae3f391b03c7b3ceb03e34f7320b0105babf48b619e7a299528b8828449e0e7696f0b5db0d99170a81709d0518e34835229a748701e7df784e58a9ce"
+		}
+	  ]
+	},
+	{
+	  "name": "imagemagick",
+	  "buildsystem": "autotools",
+	  "config-opts": [
+		"--enable-shared=no",
+	    "--enable-static=yes",
+		"--disable-docs"
+	  ],
+	  "sources": [
+	    {
+		  "type": "archive",
+		  "url": "https://download.imagemagick.org/ImageMagick/download/ImageMagick-7.1.0-32.tar.xz",
```

```suggestion
		  "url": "https://download.imagemagick.org/ImageMagick/download/releases/ImageMagick-7.1.0-32.tar.xz",
```

this is the permalink

---

Thanks, I missed that.